magellan-library/src/main/java/com/wealthfront/magellan/navigation/NavigationDelegate.kt

```diff
@@ -73,32 +72,32 @@ class NavigationDelegate(
     activity = null
   }
 
-  fun goTo(nextNavigableCompat: NavigableCompat) {
-    navigateTo(nextNavigableCompat, GO)
+  fun goTo(nextNavigableCompat: NavigableCompat, overrideTransition: Transition? = null) {
+    navigateTo(nextNavigableCompat, overrideTransition ?: DefaultTransition())
   }
 
-  fun show(nextNavigableCompat: NavigableCompat) {
-    navigateTo(nextNavigableCompat, SHOW)
+  fun show(nextNavigableCompat: NavigableCompat, overrideTransition: Transition? = null) {
+    navigateTo(nextNavigableCompat, overrideTransition ?: ShowTransition())
   }
 
-  fun replaceAndGo(nextNavigableCompat: NavigableCompat) {
-    replace(nextNavigableCompat, GO)
+  fun replaceAndGo(nextNavigableCompat: NavigableCompat, overrideTransition: Transition? = null) {
+    replace(nextNavigableCompat, overrideTransition ?: DefaultTransition())
   }
 
-  fun replaceAndShow(nextNavigableCompat: NavigableCompat) {
-    replace(nextNavigableCompat, SHOW)
+  fun replaceAndShow(nextNavigableCompat: NavigableCompat, overrideTransition: Transition? = null) {
+    replace(nextNavigableCompat, overrideTransition ?: ShowTransition())
   }
 
-  private fun replace(nextNavigableCompat: NavigableCompat, navType: NavigationType) {
+  private fun replace(nextNavigableCompat: NavigableCompat, overrideTransition: Transition? = null) {
     navigate(FORWARD) { backStack ->
       backStack.pop()
-      backStack.push(NavigationEvent(nextNavigableCompat, navType))
+      backStack.push(NavigationEvent(nextNavigableCompat, overrideTransition ?: ShowTransition()))
     }
   }
 
-  private fun navigateTo(nextNavigableCompat: NavigableCompat, navType: NavigationType) {
+  private fun navigateTo(nextNavigableCompat: NavigableCompat, overrideTransition: Transition? = null) {
```

Comment:
Same as above, this is now equivalent to `goTo()` and close to `show()`

Reply:
done